composer.json

```diff
@@ -12,7 +12,8 @@
 	"require": {
 		"php": "~7.1",
 		"phpstan/phpstan": "^0.11",
-		"nikic/php-parser": "^4.0"
+		"nikic/php-parser": "^4.0",
+		"doctrine/persistence": "^1.1"
```

Comment:
I'd like to avoid this but cannot because of https://github.com/phpstan/phpstan-doctrine/pull/49/files#diff-eaa16784d6c09802442330223866b66dR33

Any ideas? `phpcs` wont let me not have a return type for the function...

Reply:
You can use `@phpcsSuppress` annotation like `@phpcsSuppress SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingParameterTypeHint` (but with the right error code obviously)